geomstats/spd_matrices_space.py

```diff
@@ -18,12 +18,25 @@
 
 def is_symmetric(mat, tolerance=TOLERANCE):
     """Check if a matrix is symmetric."""
-    return np.allclose(mat, mat.transpose(), atol=tolerance)
+    if mat.ndim == 2:
+        mat = np.expand_dims(mat, axis=0)
+    assert mat.ndim == 3
+    n_mats, _, _ = mat.shape
+
+    mask = np.zeros(n_mats, dtype=bool)
+    for i in range(n_mats):
+        mask[i] = np.allclose(mat[i], np.transpose(mat[i]),
+                              atol=tolerance)
+
+    return mask
 
```

Comment:
maybe mask doesn’t feel like the best variable name here. Also can’t you get rid of the for loop here by doing the allclose along the first axis or something ?

Reply:
I changed mask to is_symmetric_bool. 

Yes I wanted to get rid of the for loop, but there's no axis parameter in allclose and I didn't want to hack something.